src/OnboardingSPA/components/NewfoldInterfaceSkeleton/SiteBuild/index.js

```diff
@@ -99,6 +101,7 @@ const SiteBuild = () => {
 				initialize: select( nfdOnboardingStore ).getInitialize(),
 				pluginInstallHash:
 					select( nfdOnboardingStore ).getPluginInstallHash(),
+				currentRoute: select( nfdOnboardingStore ).getCurrentStepPath(),
```

Comment:
Is there a reason why `currentStep` is failing? `currentStep` is a better check than `currentRoute` because it matches the entire `Step` class object. Same comment for wherever we have used `currentRoute` instead of `currentStep`.